cmsl1t/plotting/resolution.py

```diff
@@ -49,30 +53,35 @@ def draw(self, with_fits=False):
         fits = []
         for (pile_up, ), hist in self.plots.flat_items_all():
             if pile_up == bn.Base.everything:
-                hist.linestyle = "dashed"
-                label = "Everything"
+                # hist.linestyle = "dashed"
+                hist.drawstyle = ResolutionPlot.drawstyle
+                label = "All PU"
             elif isinstance(pile_up, int):
-                hist.drawstyle = "EP"
-                label = "~ {:.0f}".format(
-                    self.pileup_bins.get_bin_center(pile_up))
+                hist.drawstyle = ResolutionPlot.drawstyle
+                if self.pileup_bins.get_bin_upper(pile_up) < 500:
+                    label = "{:.0f} \\leq PU < {:.0f}".format(
+                        self.pileup_bins.get_bin_lower(pile_up),
+                        self.pileup_bins.get_bin_upper(pile_up),
+                    )
+                else:
+                    label = "{:.0f} < PU".format(
+                        self.pileup_bins.get_bin_lower(pile_up))
             else:
                 continue
             hist.SetMarkerSize(0.5)
-            hist.SetLineWidth(2)
+            hist.SetLineWidth(1)
             hists.append(hist)
             labels.append(label)
-            # if with_fits:
-            #     fits.append(self.fits.get_bin_contents([pile_up]))
+            if with_fits:
+                fits.append(self.fits.get_bin_contents([pile_up]))
         self.__make_overlay(hists, fits, labels, "Number of events")
 
-        integral = hist.integral()
-        normed_hists = [hist / integral if integral !=
-                        0 else hist.Clone() for hist in hists]
+        normed_hists = [hist.Clone() for hist in hists]
         for hist in normed_hists:
             if hist.integral != 0:
+                hist = hist / hist.integral()
```

Comment:
There's something strange here in that `integral` seems to be working as both a function and an attribute on `hist`.  If it's really a function, then I suppose line 81 will always return true.  I think rootpy might be defining `integral` as a getter property, in which case I dont think calling it as a function will work.  Not sure, but it's a bit confusing to me at least.